bin/build.js

```diff
@@ -1,41 +1,60 @@
-const fs = require('fs')
-const path = require('path')
-const kebabCase = require('lodash.kebabcase')
-const { version } = require('../package.json')
-const primitives = require('..')
+const fs = require("fs");
+const path = require("path");
+const kebabCase = require("lodash.kebabcase");
+const { version } = require("../package.json");
+const primitives = require("..");
 
 fs.writeFileSync(
-  path.resolve(process.cwd(), 'core-primitives.less'),
+  path.resolve(process.cwd(), "core-primitives.less"),
   [
-    `// Core Primitives v${version}`,
-    '// GENERATED FILE. DO NOT EDIT.',
+    `// Core Primitives v${version}
+// GENERATED FILE. DO NOT EDIT.`,
     toLess(primitives),
   ]
-    .map(string => string + '\n')
-    .join(''),
-)
+    .map((string) => string + "\n")
+    .join("")
+);
 
 function toLess(obj) {
   return Object.entries(flatten(obj))
     .map(([key, value]) => `@${key}: ${value};`)
-    .join('\n')
+    .join("\n");
+}
+
+fs.writeFileSync(
+  path.resolve(process.cwd(), "core-primitives.css"),
+  [
+    `/* Core Primitives v${version} */
+/* GENERATED FILE. DO NOT EDIT. */`,
+    `:root {`,
+    toCssVars(primitives),
+    `}`,
+  ]
+    .map((string) => string + "\n")
+    .join("")
```

Comment:
(minor) I think this can be refactored to:
```suggestion
    .join("\n")
```